Assets/MRTK/Services/InputSystem/FocusProvider.cs

```diff
@@ -341,9 +341,6 @@ public void UpdateHit(PointerHitResult hitResult)
                 if (hitResult.hitObject != CurrentPointerTarget)
                 {
                     Pointer.OnPreCurrentPointerTargetChange();
-
-                    // Set to default:
-                    Pointer.IsTargetPositionLockedOnFocusLock = true;
```

Comment:
I think removing this means we might want to double check any component that assumes things about this value. Looks like most of them cache the incoming value and re-set it after the interaction, but there's at least one that just blindly sets it back to `true`:

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/18fe32fd8d627216754885c7fecf4cf2434aa07d/Assets/MRTK/SDK/Experimental/ScrollingObjectCollection/ScrollingObjectCollection.cs#L2082

Reply:
Yeah I looked at this instance and saw it was an experimental feature. I changed its behavior to more closely mimic the HandInteractionPanZoom and the CanvasUtility and cache the previous set value.